instrumentation/executors/README.md

```diff
@@ -0,0 +1,6 @@
+# Settings for the executors instrumentation
+
+| System property | Type | Default | Description |
+|---|---|---|---|
+| `otel.instrumentation.executors.include` | Boolean | `false` | Whether to include property names. |
+| `otel.instrumentation.executors.include-all` | Boolean | `false` | Whether to include both executors and prefixes. |
```

Comment:
This configuration instructs us to instrument ALL classes that implement `Executor` interface, not some fixed list of known implementors.

Reply:
Fixed.